src/figuresClasses.ts

```diff
@@ -2,10 +1,79 @@
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sides = [a, b, c];
+    const maxSide = Math.max(a, b, c);
```

Comment:
Similar to the previous comment, use `this.a`, `this.b`, and `this.c` to access the properties of the class.